tours/src/main/res/layout/cell_tour_details_stop.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingTop="@dimen/marginOneHalf"
+    android:paddingBottom="@dimen/marginOneHalf">
+
+    <ImageView
+        android:id="@+id/image"
+        android:layout_width="72dp"
+        android:layout_height="45dp"
+        android:contentDescription="@null" />
+
+    <TextView
+        android:id="@+id/tourNumber"
+        android:layout_width="20dp"
```

Comment:
? why are we setting hardcoded width and height on a TextView? to make it match each other?

Reply:
to make it match designs. as the design is difficult to figure out